src/PAK-Samples/MapReferenceToPaper/App.config

```diff
@@ -1,6 +1,30 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2"/>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
```

Comment:
>NETFramework,Version=v4.6.2 [](start = 47, length = 27)

any way we can support .net Core for this App?

Reply:
I am distributing the sample using 4.6.2, users can upgrade/downgrade to whatever flavor of .NET they'd like.

The core code isn't doing anything particularly platform specific, sans HTTP

---
In reply to: [398908775](https://github.com/microsoft/mag-covid19-research-examples/pull/13#discussion_r398908775) [](ancestors = 398908775)